rpc/query_server.go

```diff
@@ -599,6 +614,35 @@ func (s *QueryServer) GetContractRecord(contractAddrStr string) (*types.Contract
 	return k, nil
 }
 
+func (s *QueryServer) DposTotalStaked() (*types.DposTotalStakedResponse, error) {
+	snapshot := s.StateProvider.ReadOnlyState()
+	defer snapshot.Release()
+	if s.totalStakedAmount != nil {
+		if time.Since(s.totalStakedAmount.createAt) <= time.Minute {
```

Comment:
Should probably make the cache interval configurable via loom.yml. The amount is only going to change when an election runs right?

Reply:
No, the amount changes all the time from the delegation, undelegation, and reward claims. I can make it configurable via loom.yaml. However, since we are using load-balancer for routing requests, the amount might be going up and down depending on the cache of nodes. We should consider cachong it on CDN instead.

Reply:
Well, if the amount is going to change all the time anyway it doesn't really matter if nodes are returning different values at a particular point in time.